MSAL/src/instance/MSALAdfsAuthorityResolver.m

```diff
@@ -207,15 +215,25 @@ - (void)webFingerRequestForEndpoint:(NSString *)authenticationEndpoint
     CHECK_ERROR_COMPLETION(authenticationEndpoint, context, MSALErrorInvalidParameter, @"AuthenticationEndpoint cannot be nil.");
     CHECK_ERROR_COMPLETION(authority, context, MSALErrorInvalidParameter, @"authority cannot be nil.");
     
-    MSALWebAuthRequest *request =
-    [[MSALWebAuthRequest alloc] initWithURL:[MSALAdfsAuthorityResolver urlForWebFinger:authenticationEndpoint absoluteAuthority:authority.absoluteString]
-                                    context:context];
-    
-    [request sendGet:^(MSALHttpResponse *response, NSError *error) {
+    NSURL *url = [MSALAdfsAuthorityResolver urlForWebFinger:authenticationEndpoint absoluteAuthority:authority.absoluteString];
```

Comment:
nit: NSURL *url to be NSURL *webfingerUrl?

Reply:
changed.